source/includes/_relationships.md

```diff
@@ -0,0 +1,69 @@
+# Relationships
+```shell
+# EXAMPLE
+GET https://api.prospect.io/public/v1/prospects/?include=organization
```

Comment:
Is this URL correct? Shouldn't it be `https://api.prospect.io/public/v1/prospects?include=organization`, without trailing `/`?